src/components/constants/index.js

```diff
@@ -0,0 +1,30 @@
+const descriptionValue = `Excepteur pariatur nulla in ulla
+mco aliqua ea cupidatat incididunt proident culpa sint non exercitation deserunt dolor. 
+Voluptate voluptate dolor reprehenderit laboris fugiat laboris consequat id ex ullamco ad duis elit. 
+Irure do sint sint do veniam aliqua consequat est deserunt ullamco. 
+Minim ut commodo exercitation deserunt sunt qui proident culpa ad ex tempor aliqua do irure. 
+Et ullamco deserunt minim adipisicing et est dolor.
+`;
+
+const selectOptions = Object.freeze({
+    MINS_15 : { 'name': '15 Minutes', 'value': '15' }, //Value is in minutes
```

Comment:
let's make it consistent, since you have used HOUR and DAY, name it as MIN_15, MIN_30

Reply:
Done